### ISSUE TITLE
Summary commands should be run on series

### ISSUE THREAD

**Issue by driskerr** [mrmcafee/intro-data-capstone-biodiversity#1]
https://github.com/mrmcafee/intro-data-capstone-biodiversity/blob/8d7f5af8152a85f1e44e6277817f673fc843f136/Codecademy/biodiversity.py#L54

Here you tried to execute: `species.nunique()`. However, `species` is a dataframe made up of multiple columns (or series). The appropariate way to determine the number of unique species in the dataset is to run `.nunique()` on the column that contains the species information (which has a "series" datatype).

So what we're looking for is `species.scientific_name.nunique()`

**Comment by mrmcafee** [comment]
Thank you, Kerry. That makes sense. I have also read through the other 7 comments and found them to be very helpful. Do I need to make these corrections and resubmit the material? Or are they just for my understanding?

**Comment by driskerr** [comment]
Of course! Feel free to make the corrections for your own satisfaction/understanding. However, there is no need to resubmit the project.
